airgun/widgets.py

```diff
@@ -2098,7 +2098,7 @@ def fill(self, values):
 class AutoCompleteTextInput(TextInput):
     """Autocomplete Search input field, We must remove the focus from this widget after fill to
     force the auto-completion list to be hidden. Since this is a react component, calling browser
-    clear method directly on the field has no effect, thus we need to clear the field using the 
+    clear method directly on the field has no effect, thus we need to clear the field using the
```

Comment:
pre-commit fixed, I dont know why

Reply:
There's space at the end of the line. I believe it to be fixed in #528 too.

Reply:
Do I need to revert that?

Reply:
I guess pre-commit makes it harder for you to not commit that change, so I would leave it as is for now. You might need to rebase if #528 is merged first, we'll see.

Reply:
Okay make sense